build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -99,6 +99,9 @@ service Execution {
   // * `CANCELLED`: The operation was cancelled by the client. This status is
   //   only possible if the server implements the Operations API CancelOperation
   //   method, and it was called for the current execution.
+  // * `NOT_FOUND`: Due to a transient condition, an unknown operation name,
+  //   or a deletion via the Operations API DeleteOperation method, the
+  //   operation no longer exists. The client should retry.
```

Comment:
We should make it explicit what should be retried. If WaitExecution() returns this error, it's not sufficient for the client to simply issue another call to WaitExecution(). It should call Execute().